lib/models/schedule.dart

```diff
@@ -0,0 +1,60 @@
+import 'package:concordia_navigation/models/course.dart';
+import 'package:device_calendar/device_calendar.dart';
+
+class Schedule {
```

Comment:
I seems the requirement also mentions giving the next class(es), we should probably add another method that uses the current time and gives back a list of future classes. We can then for example, display the first next class in the location search if its relevant to the user.

Reply:
resolved by 15fd0d15d3862cdc755e4332c6be409ff1fda9a2